src/rendering/framebufferrenderer.cpp

```diff
@@ -87,6 +87,53 @@ namespace {
 
 namespace openspace {
 
+//============================//
+//=====  Reuse textures  =====//
+//============================//
+// Gives access to the currently NOT used pingPongTexture
```

Comment:
I'd move the comments into the functions, since they are describing more what the function does, not it's outward appearance (which already has a description in the header file)